apps/dsb-client-gateway-api/src/app/types/environment-variables.ts

```diff
@@ -120,6 +120,34 @@ export class EnvironmentVariables {
   @ValidateIf(EnvironmentVariables.isAWSSecretsManagerEnabled)
   AWS_SECRET_PREFIX = '/ddhub/';
 
+  @IsString()
+  @ValidateIf(EnvironmentVariables.isAWSSecretsManagerEnabled)
+  AWS_ACCESS_KEY_ID: string;
```

Comment:
can you add those values to `docs/*.md` with description?

Reply:
@nik-ewf Could you please inform me once description and other details of these new env vars are available in MD file so I can refer use them in my Confluence pages related to deployment guide? Thanks.